apps/server/src/shared/repo/externaltool/external-tool.repo.spec.ts

```diff
@@ -303,4 +295,56 @@ describe('ExternalToolRepo', () => {
 			});
 		});
 	});
+
+	describe('findById', () => {
+		const setup2 = async () => {
```

Comment:
this should be refactored in a later test-refactoring-ticket as you have already proposed.